docs/mkdocs.yml

```diff
@@ -0,0 +1,82 @@
+site_name: SKNNR Documentation
+repo_url: https://github.com/lemma-osu/scikit-learn-knn-regression
+repo_name: lemma-osu/scikit-learn-knn-regression
+docs_dir: pages/
+
+nav:
+  - Home: index.md
+  - Installation: installation.md
+  - Usage: usage.md
+  - "API Reference":
+    - Estimators:
+      - RawKNNRegressor: api/estimators/raw.md
+      - EuclideanKNNRegressor: api/estimators/euclidean.md
+      - MahalanobisKNNRegressor: api/estimators/mahalanobis.md
+      - GNNRegressor: api/estimators/gnn.md
+      - MSNRegressor: api/estimators/msn.md
+    - Transformers:
+      - StandardScalerWithDOF: api/transformers/standardscalerwithdof.md
+      - MahalanobisTransformer: api/transformers/mahalanobis.md
+      - CCATransformer: api/transformers/cca.md
+      - CCorATransformer: api/transformers/ccora.md
+    - Datasets:
+      - Dataset: api/datasets/dataset.md
+      - "Moscow Mountain / St. Joes": api/datasets/moscow_stjoes.md
+      - "SWO Ecoplot": api/datasets/swo_ecoplot.md
+  - Contributing: contributing.md
+
+theme: 
+  name: material
+  features: 
+    - search.suggest
+    - search.highlight
+    - navigation.instant
+    - navigation.path
+    - content.code.copy
+    - content.code.annotate
+  palette:
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      toggle:
+        icon: material/weather-night
+        name: Dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      toggle:
+        icon: material/weather-sunny
+        name: Light mode
+
+plugins:
+  - search
+  - mkdocstrings:
+      handlers:
+        python:
+          paths: [../src]
+          options:
+            show_source: false
```

Comment:
You should experiment with this option and decide which you prefer. Personally I like when docs have a link to the source code, but this seemed a little obtrusive. Maybe once the docstrings are filled out it will fit better. Or maybe there's some additional config we can adjust.